site/_layouts/doc.liquid

```diff
@@ -412,7 +412,9 @@
                       <a href="#" class="sub-menu">SCIM <i class="fal fa-chevron-{% if page.url contains "scim" %}up{% else %}down{% endif %} fa-fw"></i></a>
                       <ul>
                         <li {% if page.url == "/docs/v1/tech/core-concepts/scim.html" %}class="active"{% endif %}><a href="/docs/v1/tech/core-concepts/scim">Overview</a></li>
-                        <li {% if page.url == "/docs/v1/tech/premium-features/scim/scim-sdk.html" %}class="active"{% endif %}><a href="/docs/v1/tech/premium-features/scim/scim-sdk">SCIM-SDK Example</a></li>
+                        <li {% if page.url == "/docs/v1/tech/premium-features/scim/scim-sdk.html" %}class="active"{% endif %}><a href="/docs/v1/tech/premium-features/scim/scim-sdk">SCIM-SDK</a></li>
```

Comment:
Can we do these in alphabetical order? We aren't consistent, but that is how identity providers work.

Reply:
Sure